docs/DataProviders.md

```diff
@@ -87,13 +87,16 @@ Developers from the react-admin community have open-sourced Data Providers for m
 * **[LocalStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage)**: [marmelab/ra-data-localstorage](https://github.com/marmelab/react-admin/tree/master/packages/ra-data-localstorage)
 * **[Loopback3](https://loopback.io/lb3)**: [darthwesker/react-admin-loopback](https://github.com/darthwesker/react-admin-loopback)
 * **[Loopback4](https://loopback.io/)**: [elmaistrenko/react-admin-lb4](https://github.com/elmaistrenko/react-admin-lb4)
+* **[Loopback4 CRUD](https://loopback.io/)**: [loopback4/ra-data-lb4](https://github.com/loopback4/ra-data-lb4)
```

Comment:
you should change the initial link to point to the loopback-crud-component package instead